packages/make-styles/src/makeStyles.ts

```diff
@@ -1,53 +1,87 @@
 import { createCSSVariablesProxy } from './runtime/createCSSVariablesProxy';
-import { resolveClassesBySlots } from './runtime/resolveClassesBySlots';
+import { reduceToClassNameForSlots } from './runtime/reduceToClassNameForSlots';
 import { resolveStyleRules } from './runtime/resolveStyleRules';
-import { MakeStylesOptions, MakeStylesStyleFunctionRule, MakeStylesStyleRule, ResolvedStylesBySlots } from './types';
+import {
+  CSSClassesMapBySlot,
+  CSSRulesByBucket,
+  MakeStyles,
+  MakeStylesOptions,
+  MakeStylesStyleFunctionRule,
+  MakeStylesStyleRule,
+  StyleBucketName,
+} from './types';
+
+export type StylesBySlots<Slots extends string, Tokens> = Record<Slots, MakeStylesStyleRule<Tokens>>;
+
+/**
+ * Calls resolveStyleRules() for each slot, is also used by build time transform.
+ */
+export function resolveStyleRulesForSlots<Slots extends string, Tokens>(
```

Comment:
This is an exported function but has no UTs. Are we ok with that?
Also it would be great to describe expected input and output - either by an example here or by the UTs.

Reply:
Great suggestion, done in 6a54585